rabbit-source/src/test/java/org/springframework/cloud/stream/module/rabbit/source/RabbitSourceInvalidConfigTests.java

```diff
@@ -38,8 +38,8 @@
  */
 public class RabbitSourceInvalidConfigTests {
 
-	@Rule
-	public RabbitTestSupport rabbitAvailable;
+	@ClassRule
+	public static RabbitTestSupport rabbitAvailable = new RabbitTestSupport();
```

Comment:
Well, looking closely to this test, I'd say that we really don't need this rule here.
Our `Config` below is based only on the `RabbitSourceProperties`.

@ericbottard , you are right: it doesn't have hooks to try to access to Rabbit.

I can address that on merge.